### PR TITLE
Add control-d extension

### PR DESCRIPTION
**Pull request by httpjamesm**
## Description

[Control D](https://controld.com) is a DNS service that improves your privacy and productivity. Block unwanted content, spoof your location and browse faster. No apps to install.

This extension is an API client for Control D that allows users to check their connection status to Control D, view profiles, modify active DNS filters, and authorize new IP addresses to their account.

## Screencast

change-filters
modify-profile
status

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by httpjamesm** [comment]
@peduarte Hi, sorry for the ping, but I just wanted to request a review in case you didn't see this PR already.

**Comment by peduarte** [comment]
Ok, a test API key would be great so I can I can review the extension

**Comment by peduarte** [comment]
Ok. I created an account, generated a `write` API key and loaded the extension.

I looks like I have a Blank Profile by default. The first thing I did was go to "Modify Profile" Command, and changed the name, but I got an error

image

The second thing I tried to do was go to "Change Filters", but also got an error:

(ಠ ›ಠ) — @peduarte — Mfbt95z6001714@2x

Am I doing something wrong?

**Comment by peduarte** [comment]
Activated with the promo code, seems to be working fine. Thanks for that.

File: extensions/control-d/assets/command-icon.png
Comment (peduarte) [inline extensions/control-d/assets/command-icon.png:1]: Can you make sure the extension icon follows [the guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store#extension-icon) please? Most notable the border radius and the size of the icon within the bounding box. In the guidelines you'll find a Figma template with guides. 

Alternatively, you can use https://icon.ray.so/ and upload the logo, choose a black background, and export

**Comment by peduarte** [comment]
Looks great, good work. Small comment about the icon and we can publish 🚀

**Comment by httpjamesm** [comment]
@peduarte Fixed!

**Review by peduarte — APPROVED** [review]
Looks great, congrats!

**Pull request merged**